packages/devtools_app/lib/src/timeline/flutter/timeline_screen.dart

```diff
@@ -32,21 +32,33 @@ class TimelineScreen extends Screen {
   }
 }
 
-class TimelineBody extends StatefulWidget {
-  const TimelineBody();
+class TimelineScreenBody extends StatefulWidget {
+  final controller = TimelineController();
```

Comment:
Every time this widget gets rebuilt, it will create a new controller.  If anything in the controller needs to be disposed, it won't be.

Based on our discussion yesterday, I think the reason for putting this here was to make it available to the test.  There's another option in the test code: accessing the `tester.state` instead of the widget.

```dart
final body = TimelineScreenBody();
tester.pumpWidget(wrap(body));
final state = tester.state(find.byWidget(body)) as TimelineScreenBodyState;
expect(state.controller.foo, bar);
```